Update cpesearch example (#715)
getsolus/help-center-docs: docs/packaging/monitoring.yaml.md

```text
[
  [
    916.0,
    "cpe:2.3:a:systemd_project:systemd"
  ],
  [
    1.0,
    "cpe:2.3:a:ubuntu_developers:systemd"
  ]
]
Verify successful hits by visiting https://cve.circl.lu/search/$VENDOR/$PRODUCT
- CPE entries for software applications have the form 'cpe:2.3:a:$VENDOR:$PRODUCT'

```

Ignore the numerical ids, let's walk through the CPEs by vendor:

- `ubuntu_developers` is for `systemd` patched by Ubuntu; we can ignore it
- `systemd_project` is a good candidate, so we add it

> If you see something like `"cpe:2.3:a:lennart_poettering:systemd"`, the vendor is Lennart Poettering and this shouldn't be used. This is a bleeding edge / development vendor, rather than production.

Reviewer: ```suggestion
If you see something like `"cpe:2.3:a:lennart_poettering:systemd"`, the vendor is Lennart Poettering and this shouldn't be used. This is a bleeding edge / development vendor, not the actual production vendor.
```
